catwalk/tasks/__init__.py

```diff
@@ -169,5 +193,27 @@
         "wnli",
         "wsc",
     },
-    "raft": {name for name in TASKS.keys() if name.startswith("raft::")}
+    "raft": {name for name in TASKS.keys() if name.startswith("raft::")},
+    "metaicl-classification-eval": {
+        "metaicl::tweet_eval-stance_feminist",
+        "metaicl::ethos-national_origin",
+        "metaicl::tweet_eval-hate",
+        "metaicl::ag_news",
+        "metaicl::amazon_polarity",
+        "metaicl::hate_speech18",
+        "metaicl::poem_sentiment",
+        "metaicl::climate_fever",
+        "metaicl::medical_questions_pairs",
+        "metaicl::tweet_eval-stance_atheism",
+        "metaicl::superglue-cb",
+        "metaicl::dbpedia_14",
+        "metaicl::wiki_qa",
+        "metaicl::emo",
+        "metaicl::yelp_polarity",
+        "metaicl::ethos-religion",
+        "metaicl::financial_phrasebank",
+        "metaicl::tab_fact",
+        "metaicl::anli",
+        "metaicl::ethos-race"
+    },
```

Comment:
You didn't want to do the same `if name.startswith(...)` thing?

Reply:
That would be better! I'll change it

Reply:
This isn't actually all of the datasets in metaicl. I loaded in piqa and boolq as well which are from the _training_  part of the classification tasks in the benchmark. Eventually I will have all the tasks, and then there can be a separate `"metaicl-classification-train"` and so on if desirable.